code/lib/core-server/src/utils/StoryIndexGenerator.ts

```diff
@@ -486,10 +491,17 @@ export class StoryIndexGenerator {
         });
       });
 
-      const notFound = dependents.filter((dep) => !invalidated.has(dep));
-      if (notFound.length > 0) {
-        throw new Error(`Could not invalidate ${notFound.length} deps: ${notFound.join(', ')}`);
-      }
+      /**
+       * Currently, we don't sort cacheEntries and invalidate cacheEntries from type 'stories' first, but
+       * cacheEntries with type 'docs' might be invalidated before ones with cacheEntry type 'story'.
+       * This can cause a race condition, because cacheEntries from type 'docs' might already be deleted,
+       * before cacheEntries from type 'story' are invalidated.
+       * Therefore, throwing an error here is not a good idea.
+       */
+      // const notFound = dependents.filter((dep) => !invalidated.has(dep));
+      // if (notFound.length > 0) {
+      //   throw new Error(`Could not invalidate ${notFound.length} deps: ${notFound.join(', ')}`);
+      // }
```

Comment:
OK, this makes sense, I guess we drop this code?

Reply:
Sure! Just wanted to know your opinion about this first.